rasa/core/events/__init__.py

```diff
@@ -180,19 +180,21 @@ class UserUttered(Event):
 
     def __init__(
         self,
-        text,
+        text: Text = None,
```

Comment:
I think this should not have a default value.
```suggestion
        text: Text,
```